src/import-service/serverless.ts

```diff
@@ -62,6 +62,13 @@ const serverlessConfiguration: Serverless = {
               }
             },
             cors: true,
+            authorizer: {
+              name: 'tokenBasicAuthorizer',
+              arn: 'arn:aws:lambda:eu-west-1:549933849673:function:authorization-service-dev-basicAuthorizer',
```

Comment:
Would be better to add BasicAuthorizer function to the Outputs. Cause afaik arn will be changed if you deploy some changes to he function